_posts/2023-02-24-puppetmodule.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Make puppet module documentation great again!
+date: 2023-02-24
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+A long long time ago I can still remember... [Dominic Cleal](https://github.com/domcleal)
+created [puppetmodule.info](https://www.puppetmodule.info/). Inspired by
+[rubydoc.info](https://rubydoc.info/), the site generates documentation for
+Puppet modules with [puppet-strings](). puppetmodule.info was created around
+2015. Since then Dominic moved on and isn't active anymore in the Puppet
+ecosystem and the site died.
```

Comment:
According to the whois, the domain was created 2017-01-16T16:02:39Z (but maybe it has been created before, expired, and re-created).

Does a website "die"?  Was in not broken, or just left untouched/abandoned?  It feels awkward to me, but I am not native English speaker so maybe it is very fine :smile:

Reply:
"die" as in: the code wasnt touched for years, dominic forgot that people used it and then heroku deleted it because they disabled their free tier :D

Reply:
I updated the phrasing a bit. Indeed it was 2017.